src/python/turicreate/toolkits/one_shot_object_detector/one_shot_object_detector.py

```diff
@@ -296,6 +295,20 @@ def __repr__(self):
             class_name='OneShotObjectDetector')
         return out
 
+    def summary(self, output=None):
+
+        if output is None or output == 'stdout':
```

Comment:
This code appears to be an exact copy of the code in the base class. Am I missing some thing?

If this code is an exact copy of what's in the base class, I don't understand why we're overriding it.

Reply:
For OSOD, the summary is actually the OD model,
so instead of doing 
```
return _toolkit_serialize_summary_struct( self, *self._get_summary_struct() )
```
Here is 
```
return _toolkit_serialize_summary_struct( self.__proxy__['detector'],*self._get_summary_struct() )
```